prow/config/jobs_test.go

```diff
@@ -777,34 +766,18 @@ func TestMergePreset(t *testing.T) {
 		},
 	}
 	for _, tc := range tcs {
-		conf := &Config{
-			Presets: tc.presets,
-			Periodics: []Periodic{
-				{
-					Name:     "foo",
-					Labels:   tc.jobLabels,
-					Agent:    string(kube.JenkinsAgent),
-					Interval: "1h",
-					Spec:     tc.pod,
-				},
-			},
-		}
-		if err := parseConfig(conf); err == nil && tc.shouldError {
```

Comment:
this was a little roundabout for what we really wanted to test, had to fix since the `PodSpec`s in the tests above were not valid